### ISSUE TITLE
About the root joint's position

### ISSUE THREAD

**Issue by hellozjj** [facebookresearch/VideoPose3D#130]
I see in your readme file there is a case that the root joint is also moving.
Does the prediction include the root joint's world position, not just to fix it to (0,0,0)?

**Comment by PKUFlyingPig** [comment]
I just found the same problem, do you have some idea about that?

**Comment by jionie** [comment]
I have tried to retrain a model without semi-supervise learning, but I found the model doesn't learn any position info for root node, the loss backward in training codes doesn't include the root node info `inputs_3d[:, :, 0] = 0 ` `loss_3d_pos = mpjpe(predicted_3d_pos, inputs_3d)`.
